### PR TITLE
big-o-notation - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**

### CODE REVIEW

File: big-o-notation/big-o.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  for (
+    let i = 0; // 1 * 1 = O(1)
+    i < words.length; // 2 * n = O(2n)
+    i++ // 2 * n = O(2n)
+  ) {
+    const word = words[i]; // 2 * n = O(2n)
+    if (!seen[word]) {
+      // 2 * n = O(2n)
+      seen[word] = true; // 2 * n = O(3n)
+      unique[unique.length] = word; // 3 * n = O(3n)
```
Comment (shawnkost) [inline big-o-notation/big-o.js]: You want to drop all the coefficients, so it's just `O(n)`

File: big-o-notation/big-o.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  const unique = []; // 1 * 1 = O(1)
+  for (
+    let i = 0; // 1 * 1 = O(1)
+    i < words.length; // 2 * n = O(2n)
+    i++ // 2 * n = O(2n)
+  ) {
+    const word = words[i]; // 2 * n = O(2n)
+    let isUnique = true; // 2 * n = O(2n)
+    for (
+      let c = 0; // 1 * n = O(1n)
```
Comment (shawnkost) [inline big-o-notation/big-o.js]: All just `O(n)`

File: big-o-notation/big-o.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      c < i; // 1 * n * n = O(n^2)
+      c++ // 2 * n * n = O(2n^2)
+    ) {
+      const comparing = words[c]; // 2 * n * n = O(2n^2)
+      if (comparing === word) {
+        // 1 * n * n = O(1n^2)
+        isUnique = false; // 1 * n * n = O(1n^2)
```
Comment (shawnkost) [inline big-o-notation/big-o.js]: All just `O(n^2)`

File: big-o-notation/big-o.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      }
+    }
+    if (isUnique) {
+      // 1 * n = O(1n)
```
Comment (shawnkost) [inline big-o-notation/big-o.js]: Should just be `O(n)`

File: big-o-notation/big-o.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      unique[unique.length] = word; // 3 * n = O(3n))
```
Comment (shawnkost) [inline big-o-notation/big-o.js]: Should be `O(n)`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Roland!

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**